src/services/services.ts

```diff
@@ -1244,6 +1244,7 @@ namespace ts {
     }
 
     export interface EditorOptions {
+        BaseIndentSize?: number;

```

Comment:
There are default editor options that are applied for TS server so these are all defined as optional in protocol.d.ts in case the user wants to override the defaults. For TS services there are no defaults so the editor options are all required except for the new BaseIndentSize—optional for hosts that want to support base indentation. These two shapes (including differences in case) are reconciled in editorServices.mergeFormatOptions.  @vladima mentioned he is going to make the case consistent with his changes to ts server.